webview-ui/src/CreateCluster/CreateCluster.module.css

```diff
@@ -2,69 +2,66 @@
     display: grid;
     grid-template-columns: 8rem 20rem 8rem;
     grid-gap: 1rem;
-    margin-top: 18px;
 }
+
 .presetHeader {
-    width: 250px;
-    height: 18px;
-    margin-bottom: 10px;
+    height: 1rem;
+    margin: 0 0 0.6rem 0;
+    width: 15rem;
 }
 
 .presetContainer {
-    width: 217px;
-    height: 166px;
-    flex-shrink: 0;
-    border-radius: 8px;
-    border: 1px solid var(--base-base-10, #808080);
-    margin-bottom: 23px;
+    border: 0.0625rem solid var(--vscode-editorCursor-foreground);
```

Comment:
For this one in particular, I'd want to do a quick test of switching between dark mode and light mode, and checking it looks good in both cases.

Reply:
Removed the color from most of the css to make it work for all vscode color themes. check the video.